experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -88,13 +80,12 @@ const defaultEditGeneratorOpWeights: EditGeneratorOpWeights = {
 	start: 0,
 	commit: 0,
 	abort: 0,
-	synchronize: 0,
 };
 
 export const makeNodeEditGenerator = (): Generator<NodeEdit, EditState> => {
 	function setPayloadGenerator(state: EditState): FuzzNodeEditChange {
-		const trees = state.trees;
-		const tree = trees[state.treeIndex];
+		const trees = state.clients;
+		const tree = trees[state.treeIndex].channel;
```

Comment:
to illustrate point above:

```suggestion
		const tree = state.channel;
```

Reply:
replaced with `state.channel`